packages/cli/src/config/snippets.ts

```diff
@@ -0,0 +1,5 @@
+const BASE_SNIPPETS_URL = "https://snippet.meticulous.ai/";
+
+export const getSnippetsBaseUrl = (): string => {
+  return process.env["METICULOUS_SNIPPETS_BASE_URL"] || BASE_SNIPPETS_URL;
+};
```

Comment:
To validate `METICULOUS_SNIPPETS_BASE_URL`, I would do:

```suggestion
export const getSnippetsBaseUrl = (): string => {
  try {
    return new URL(process.env["METICULOUS_SNIPPETS_BASE_URL"] || BASE_SNIPPETS_URL).href;
  } catch (error) {
    if (error instanceof TypeError) {
      return BASE_SNIPPETS_URL;
    }
    throw error;
  }
};
```

And maybe also memoize the value? Not needed to merge though :smile:

Reply:
I'd err on the side of loudly erroring rather than swallowing the error here and defaulting, https://github.com/alwaysmeticulous/meticulous-sdk/pull/84/commits/384411ab2396db7f9f042461a60833152d2743f9 makes the validation error bit clear though.

Reply:
That works too! :ok_hand: